CONTRIBUTING.md

```diff
@@ -1,10 +1,72 @@
-Submit a pull request!
-
 # Development environment
 
-The [librivox-ansible](https://github.com/LibriVox/librivox-ansible) playbooks
-can be used to set up a local development server. It comes with a [database
-snapshot](https://github.com/LibriVox/librivox-ansible/blob/master/resources/librivox_catalog_scrubbed.sql.bz2)
-that has been
-[scrubbed](https://github.com/LibriVox/librivox-ansible/blob/master/dbscrub.py)
-of all personal information.
+The [librivox-ansible] playbooks can be used to set up a local development
+server. It comes with a [database snapshot] that has been
```

Comment:
s/It comes/They come/

I know you didn't actually write this, but if we're updating this, could you fix this small nit as well please?